index.html

```diff
@@ -75,108 +81,123 @@ <h1 class="primary-title font-family-poppins">I'm Belden.<br> Glad to see you!</
-               <li class="column-link"><a href="#">css</a></li>
-               <li class="column-link"><a href="#">javascript</a></li>
-            </ul>
-
+            <div class="column-detail">
+                <h2 class="col-title">Facebook 360</h2>
+    
+                <ul class="canopy">
+                   <li class="canopy-title ">facebook</li>
+                   <li class="can-items">
+                     <span>• </span>  Full Stack Dev <span>• </span>
+                   </li>
+                   <li class="can-items">2015</li>
+                </ul>
+    
+                <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+    
+                <ul class="column-links">
+                   <li class="column-link"><a href="#">html</a></li>
+                   <li class="column-link"><a href="#">ruby on rails</a></li>
+                   <li class="column-link"><a href="#">css</a></li>
+                   <li class="column-link"><a href="#">javascript</a></li>
+                </ul>
+    
+    
+                <a href="" class="column-button"> see project</a>
 
-            <a href="" class="column-button"> see project</a>
+            </div>
          </div>
 
-         <div class="col-1 column-item">
+         <div class="col-1 column-item row-reverse">
             <img src="images\Snapshoot (1).jpg" alt="">
-            <h2 class="col-title">Multi-post <br> stories</h2>
+            <div class="column-detail">
+                
+            <h2 class="col-title">Uber Navigation</h2>
 
             <ul class="canopy">
-               <li class="canopy-title ">canopy</li>
+               <li class="canopy-title ">Uber</li>
                <li class="can-items">
-                 <span>• </span>  backend-dev <span>• </span>
+                 <span>• </span>  Lead Developer <span>• </span>
                </li>
-               <li class="can-items">2015</li>
+               <li class="can-items">2018</li>
             </ul>
 
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
 
             <ul class="column-links">
                <li class="column-link"><a href="#">html</a></li>
+               <li class="column-link"><a href="#">ruby on rails</a></li>
                <li class="column-link"><a href="#">css</a></li>
                <li class="column-link"><a href="#">javascript</a></li>
             </ul>
 
 
             <a href="" class="column-button">see project</a>
+            </div>
          </div>
 
            
         </section>
 
         <section class="about-me font-family-poppins">
            
-            <div class="about-details">
+            <div  class="about-details">
```

Comment:
Well done styling the about section 🥇
- [ ] I think it would be better if you style you button according to the figma template. What do you think ? 🎯😄 I can see that you have done very well with other buttons, let's make sure to add the correct `background` to the about section button as well . I know you can do this. 💪
- [ ] Also, please make sure that the background color of `languages` icon is not white but according to what is given in figma template in order to follow the project requirements. You got this 🎯
- [ ] What do you think of removing the border after `languages` as it is not present in figma template. I think it would be better if you consider removing it. What do you think ? 💪

## **_Your design 👇_**
![image](https://user-images.githubusercontent.com/106119105/214616168-cb49bf59-0489-42d1-be4c-bbdaa8db26b4.png)

## **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/214615780-96260333-7deb-42ca-9118-2960f4572bf4.png)